### PR TITLE
Switch to multiarch/alpine

### PR DESCRIPTION
**Pull request by jedahan**
This PR switches from ubuntu to alpine, and has COPY_DIR merge into root instead of create a subdirectory, and documents how to customize your images.

It needs testing on more than just my linux server and macOS laptop.

The good:
```zsh
~/ docker images pi-maker && docker images alpi-maker | tail -n1
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
pi-maker            latest              5c46bb8c5817        21 hours ago        1.05GB
alpi-maker          latest              0bc01378bb21        6 minutes ago       55.7MB
```

Having pi-maker based off of multiarch I think means we also get rid of the binfmt and qemu install dependencies since we just use `docker run --rm --privileged multiarch/qemu-user-static:register --reset` but I'm not totally sure about that. This is pretty new territory for me.

Let me know your thoughts!

### CODE REVIEW

**Comment by westlane** [comment]
I am trying out the new alpine code and look forward to the possibility of switching to this approach. I did have to replace "install" with "add" in the Dockerfile for apk, since "install" command was not recognized. After doing so I am able to properly build the docker image. However, the run script appears to stall out when trying to unpack the Arch Linux file. Have you experienced anything similar to this?

script paused

**Comment by jedahan** [comment]
So I changed around some of the one-liners in extracting which should fix the hanging in bash, and I added a progress bar to untarring so it doesn't seem like its hanging.

svg of it working on my machine:
[termtosvg_cyxpo_tv.svg.zip](https://github.com/lantern-works/pi-maker/files/2673455/termtosvg_cyxpo_tv.svg.zip)

File: bin/disk-init [hunk 1]
```diff
@@ -23,7 +23,7 @@ if [[ ! $(ls "${loopdev}"*1) ]]; then
     echo "WARNING: Missing a required partition. This can happen once."
     echo "Please run script once more to fix."
     echo "------------------------------------------"
-    kill 0
+    exit
```
Comment (jedahan) [inline bin/disk-init]: So one thing that worries me - does this exit keep a running container around? Whereas kill 0 does not?

Comment (jedahan) [reply bin/disk-init]: Also is this missing loopback partition issue documented anywhere else?

**Pull request merged**